mlos_bench/mlos_bench/environments/base_environment.py

```diff
@@ -152,8 +168,10 @@ def __init__(  # pylint: disable=too-many-arguments
         req_args = set(config.get("required_args", [])) - set(
             self._tunable_params.get_param_values().keys()
         )
```

Comment:
btw there's no need for second `set()` cast:
```suggestion
        req_args = set(config.get("required_args", [])) - self._tunable_params.get_param_values().keys()
```